.github/workflows/audit.yml

```diff
@@ -0,0 +1,15 @@
+name: Security audit
```

Comment:
The README of this repo suggests about specifying `paths` to speed things up (https://github.com/actions-rs/audit-check#audit-changes). Is this something you think would matter here?